cypress/integration/onboarding/multi_team/existing_channel_name_spec.js

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [number] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+/*eslint max-nested-callbacks: ["error", 3]*/
+
+/*
+function: verifyExistingChannelError
+arg - channelTypeID:        ID of public or private channel to create
+arg - newChannelName:       New channel name to assign
+
+This function clicks on #createPublicChannel or #createPrivateChannel.
+Types `newChannelName` provided in the #newChannelName input.
+Clicks the 'Create New Channel' button.
+Then verifies that an error message 'A channel with that name already exists on the same team' is shown.
+*/
+function verifyExistingChannelError(channelTypeID, newChannelName) {
+    // Click on '+' button for Public or Private Channel
+    cy.get(channelTypeID).click();
+
+    // Type `newChannelName` in the input field for new channel
+    cy.get('#newChannelName').type(newChannelName);
+
+    // Click 'Create New Channel' button
+    cy.get('#submitNewChannel').click();
+
+    // * User gets a message saying "A channel with that name already exists on the same team"
+    cy.get('#createChannelError').contains('A channel with that name already exists on the same team');
+}
+
+/*
+function: channelNameTest
+arg - channelTypeHeading:   PUBLIC CHANNELS or PRIVATE CHANNELS
+arg - channelName:          Existing channel name that is also being tested for error
+
+var - origChannelLength:    The original number of channels in PUBLIC CHANNELS or PRIVATE CHANNELS
+
+This function stores the number of channels in public or private channels as origChannelLength.
+Verifies that the channel name picked exists.
+Attempts creating a public channel with the existing `channelName`, which should return error as implemented in `verifyExistingChannelError`.
+Then atempts createing a private channel with the existing `channelName`, which should return error as implemented in `verifyExistingChannelError`.
```

Comment:
Spell check on `atempts createing`

Reply:
If possible, please modify the comment to have short and concise description.  Same case for `verifyExistingChannelError`

Reply:
Thank you for reviewing! Fixed the typos and shortened descriptions.